lib/runnable.js

```diff
@@ -24,8 +24,10 @@ var MAX_TIMEOUT = Math.pow(2, 31) - 1;
 
 module.exports = Runnable;
 
+// Add "Additional properties" doc comment for hosted docs :)
 /**
  * Initialize a new `Runnable` with the given `title` and callback `fn`.
+ * Additional properties, like `getFullTitle()` and `slow()`, can be viewed in the runnable source.
```

Comment:
[Docs] For consistency:

```suggestion
 * Additional properties, like `getFullTitle()` and `slow()`, can be viewed in the `Runnable` source.
```